torch/_dynamo/variables/torch_function.py

```diff
@@ -573,8 +574,13 @@ def dispatch_torch_function(tx: "InstructionTranslator", fn, args, kwargs):
         if not (isinstance(res, ConstantVariable) and res.value is NotImplemented):
             return res
 
-    unimplemented(
-        f"All __torch_function__ overrides for call {fn} with args {args} and kwargs {kwargs} returned NotImplemented"
+    unimplemented_v2(
+        gb_type="TypeError from user code",
+        context=f"{fn=}, {args=}, {kwargs=}",
+        explanation="No available `__torch_function__` override found",
```

Comment:
Maybe mention that specifically, "all __torch_function__ overrides returned NotImplemented"?

Reply:
Updated.